packages/clay-nav/src/Vertical.tsx

```diff
@@ -150,15 +163,24 @@ function renderItems(items: Array<IItem>, spritemap?: string, level = 0) {
 	});
 }
 
-export const ClayVerticalNav: React.FunctionComponent<IProps> = ({
+const ClayVerticalNav: React.FunctionComponent<IProps> & {
+	Trigger: typeof Trigger;
+} = ({
 	activeLabel,
 	items,
 	large,
 	spritemap,
+	trigger: CustomTrigger = Trigger,
```

Comment:
That's interesting, I didn't know that Typescript had the capability to basically decide what renders based on it's props like that. So if `trigger` isn't provided it will render the `<CustomTrigger />` here, and when `trigger` is provided it will still wrap it in the `ClayButton` because they share the same children; That's very powerful.

Reply:
Yeah, this is very powerful, we can eliminate some lines and conditionals. Basically an alias with a default value.

Reply:
> I didn't know that Typescript had the capability

And not even specific to TypeScript. See the "Default values" examples [on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).